python/pyspark/ml/classification.py

```diff
@@ -1511,27 +1512,47 @@ class OneVsRest(Estimator, OneVsRestParams, MLReadable, MLWritable):
     .. versionadded:: 2.0.0
     """
 
+    parallelism = Param(Params._dummy(), "parallelism",
```

Comment:
Since this is a shared Param in Scala, do you want to make it one in Python too?  You can add it to _shared_params_code_gen.py and then re-generate the shared.py file.